packages/wdio-devtools-service/src/auditor.js

```diff
@@ -94,22 +94,26 @@ export default class Auditor {
     }
 
     async getMetrics () {
-        const timeToFirstByte = await this._audit(TTFBMetric, { URL: this.url })
+        const serverResponseTime = await this._audit(ServerResponseTime, { URL: this.url })
         const result = await this._audit(Metrics)
         const metrics = result.details.items[0] || {}
         return {
             estimatedInputLatency: metrics.estimatedInputLatency,
-            timeToFirstByte: Math.round(timeToFirstByte.numericValue, 10),
+            timeToFirstByte: Math.round(serverResponseTime.numericValue, 10),
```

Comment:
```suggestion
            /**
             * keeping TTFB for backwards compatibility
             */
            timeToFirstByte: Math.round(serverResponseTime.numericValue, 10),
```

Reply:
I slept yesterday. I see you have already made the change. Thanks a lot.